src/lib/merkle_ledger/util.ml

```diff
@@ -0,0 +1,47 @@
+open Core
+
+module Make (Location : sig
+  type t
+end) (Account : sig
+  type t
+end)
```

Comment:
Generally, when we have more than three functor parameter inputs, we coalesce into one giant module, like how you see in `Transition_frontier`.

https://github.com/CodaProtocol/coda/blob/master/docs/style_guide.md

Can you make the functor parameter inputs into one module like the following:
```ocaml
module type Inputs_intf : sig
   module Location : sig type t end
   module Account : sig type t end
   ....
end


```

Reply:
The ledgers all have more than 3 parameters. It would be good to have them all in one giant Inputs module. Maybe you can do that in this PR or another one.